tensorboard/webapp/metrics/views/main_view/pinned_view_container.ts

```diff
@@ -41,4 +42,33 @@ export class PinnedViewContainer {
   readonly cardIdsWithMetadata$: Observable<
     DeepReadonly<CardIdWithMetadata[]>
   > = this.store.select(getPinnedCardsWithMetadata).pipe(startWith([]));
+
+  readonly newCardPinnedIds$: Observable<number[]> = this.store
```

Comment:
This name confused me. AFAICT it does not describe what is actually resulting from the code below. Maybe something like TimestampOfLatestPin or TimeOfLatestPinning?

Reply:
Left a little comment on what the field is supposed to be. It is supposed to be an opaque id.

Reply:
Ok thanks for the comment. That helps.